firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -27,7 +27,7 @@ import kotlinx.coroutines.CoroutineDispatcher
 
 class FirebaseSessions
 internal constructor(
-  firebaseApp: FirebaseApp,
+  val firebaseApp: FirebaseApp,
```

Comment:
Does this need to be `private val` ? Default is public in Kotlin. This class is public, but ctor is internal. I am not sure if Java will see getFirebaseApp() as a public method?

Reply:
Very good point and thanks for surfacing. You are right keeping it private is totally a good idea.